pkg/syncer/status/status_process.go

```diff
@@ -59,7 +59,7 @@ func (c *Controller) process(ctx context.Context, gvr schema.GroupVersionResourc
 	}
 	downstreamClusterName, name := clusters.SplitClusterAwareKey(clusterAwareName)
 	// TODO(sttts): do not reference the cli plugin here
-	if strings.HasPrefix(workloadcliplugin.SyncerIDPrefix, downstreamNamespace) {
+	if strings.HasPrefix(downstreamNamespace, workloadcliplugin.SyncerIDPrefix) {
```

Comment:
@sttts I think this was an actual typo introduced in #1011.

Reply:
nice, a classic :)

Reply:
Good thing the variables were named so clearly that a linter could pick this up :)